README.md

```diff
@@ -397,15 +397,15 @@ in [doc/plugins.md](doc/plugins.md#hooks).
 ### Limitations
 
 It is not possible to call Finit via signals or use `initctl` in any
-runparts or `/etc/rc.local` script.  This because Finit is single
+runparts or `/etc/rc.local` script.  This is because Finit is single
 threaded and is calling these scripts in a blocking fashion at the end
 of runlevel S, at which point the event loop has not yet been started.
 
 The event loop is the whole thing which Finit is built around, except
 for runlevel S, which remains a slow procession through a lot of set up,
 with a few hooks and blocking call outs to external scripts.
 
-However, not all `initctl` commands are prohibited, supported commands:
+However, not all `initctl` commands are prohibited, supported commands
```

Comment:
I'd rather drop 'are:'

Reply:
If we drop the last "are", then it would be better to make "Supported commands:" into its own sentence. In it current form, the two parts seem to be coordinated clauses, in which second one needs a verb phrase just as the first one. (But I wouldn't bet on it, then again, it just feels odd.)

Reply:
Yes, good point.  Please change to two separate sentences.